src/mlx_omni_server/main.py

```
@@ -4,8 +4,11 @@
 import uvicorn
 from fastapi import FastAPI
 
+from mlx_omni_server.utils.logger import set_logger_level
+
 from .middleware.logging import RequestResponseLoggingMiddleware
 from .routers import api_router
+from .utils.logger import logger, set_logger_level
```

Ruff (0.12.2)

Redefinition of unused `set_logger_level` from line 7

(F811)